projects/plugins/jetpack/modules/masterbar/inline-help/inline-help.js

```diff
@@ -0,0 +1,22 @@
+( function () {
+	var windowObjectReference = null;
+	function init() {
+		document.addEventListener( 'click', function ( e ) {
+			if ( e.target.dataset.fauxInlineHelp !== undefined ) {
+				e.preventDefault();
+
+				if ( windowObjectReference == null || windowObjectReference.closed ) {
```

Comment:
Linting issue needs fixing

```suggestion
				if ( windowObjectReference === null || windowObjectReference.closed ) {
```